custom/icds_reports/utils/aggregation_helpers/distributed/postnatal_care_forms_ccs_record.py

```diff
@@ -83,7 +83,7 @@ def aggregation_query(self):
             COALESCE(ucr.case_id, prev_month.case_id) AS case_id,
             GREATEST(ucr.latest_time_end, prev_month.latest_time_end_processed) AS latest_time_end_processed,
             GREATEST(ucr.counsel_methods, prev_month.counsel_methods) AS counsel_methods,
-            GREATEST(ucr.new_ifa_tablets_total, prev_month.new_ifa_tablets_total) AS new_ifa_tablets_total,
+            COALESCE(ucr.new_ifa_tablets_total, prev_month.new_ifa_tablets_total) AS new_ifa_tablets_total,
```

Comment:
This coalesce will always take the first value since the first value is never null according to your query above